src/org/omegat/util/gui/UIDesignManager.java

```diff
@@ -69,12 +71,47 @@
  * @author Alex Buloichik
  * @author Yu Tang
  * @author Aaron Madlon-Kay
+ * @author Hiroshi Miura
  */
 public final class UIDesignManager {
 
+    private static final Set<String> BLACKLIST = new HashSet<>();
+
+    static {
+        BLACKLIST.add("CDE/Motif");
+        BLACKLIST.add("Metal");
+        BLACKLIST.add("GTK+");
+    }
+
     private UIDesignManager() {
     }
 
+    public static List<String> getThemes() {
+        List<String> names = new ArrayList<>();
+        names.add("Default");
+        UIManager.LookAndFeelInfo[] lafInfo = UIManager.getInstalledLookAndFeels();
+        for (UIManager.LookAndFeelInfo info: lafInfo) {
+            if (!BLACKLIST.contains(info.getName())) {
+                names.add(info.getName());
```

Comment:
I see that we are using the theme "name" as the preferences value.

Is that safe? Are you sure the name is not localized? That it never changes across Java versions?

Would it not be safer to use the classname as the preference value, and show the display name for the combo box via a renderer? (See `GlossaryPreferencesController#initGui` for an example.)

Reply:
>Would it not be safer to use the classname as the preference value, and show the display name for the combo box via a renderer? 

getName() returns String for menu
https://docs.oracle.com/javase/8/docs/api/javax/swing/UIManager.LookAndFeelInfo.html#getName--

>Returns the name of the look and feel in a form suitable for a menu or other presentation

In Java 16, it is the same.
https://docs.oracle.com/en/java/javase/16/docs/api/java.desktop/javax/swing/UIManager.LookAndFeelInfo.html#getName()

Reply:
I found  toString() is safer to save preference as identifier 
https://docs.oracle.com/en/java/javase/16/docs/api/java.desktop/javax/swing/UIManager.LookAndFeelInfo.html#toString()

Reply:
Now use `LookAndFeelInfo.toString()` as a key.